build.gradle.kts

```diff
@@ -110,7 +115,9 @@ configurations {
 sourceSets {
     compatTest {
         compileClasspath += sourceSets["test"].output
+        compileClasspath += sourceSets["main"].output
```

Comment:
Do you mean `.output.classesDirs` here?

Reply:
The above works. Probably there is an overloaded method to support it.

Reply:
`.output` is more than `classes`. it includes resources as well.

Do you really need to include `resources` on the **compile**  classpath?

Reply:
To compile not, but in runtime it might be necessary in some cases. E.g. with Nebula Test it would be required to provide a properties file with the main plugin class provided (with a plugin for plugins and TestKit is not necessary, however).